test/integration/integration-test.js

```diff
@@ -147,10 +175,145 @@ describe.skip('integration tests', function() {
     ], () => done());
   });
 
+
+  it('isconnected', function(done) {
+    assert(this.api.isConnected());
+    done();
+  });
+
+
+  it('serverinfo', function(done) {
+    this.api.getServerInfo(composeAsync(data => {
+      assert(data && data.info && data.info.pubkey_node);
+    }, done));
+  });
+
+
+  it('getfee', function(done) {
+    const fee = this.api.getFee();
+    assert.strictEqual(typeof fee, 'string');
+    done();
+  });
+
+
+  it('getledgerversion', function(done) {
+    const ledgerVersion = this.api.getLedgerVersion();
+    assert.strictEqual(typeof ledgerVersion, 'number');
+    done();
+  });
+
+
+  it('gettransactions', function(done) {
+    const options = {
+      outgoing: true,
+      minLedgerVersion: this.startLedgerVersion
+    };
+    this.api.getTransactions(wallet.getAddress(), options, composeAsync(transactionsData => {
+      assert(transactionsData);
+      assert.strictEqual(transactionsData.length, this.transactions.length);
```

Comment:
Yeah, they should return the hash. The hash is not really an "outcome", so maybe we should add a "properties" section to the response. Do you want to add that?